tests/dummy/app/pods/application/template.hbs

```diff
@@ -1,9 +1,6 @@
 {{frost-select-outlet}}
-
 <div class="dummy-notification-container">
-  {{#each notifications as |notification|}}
-    {{notification-message notification=notification}}
-  {{/each}}
+{{notification-container position='top-right'}}
```

Comment:
I wonder if we should change `notification-container` to `notification-outlet` in `ember-frost-notification` to match the naming of `frost-select-outlet` and `frost-modal-outlet`.